server/models/image_metadata.py

```diff
@@ -20,56 +20,113 @@
 import six
 
 
```

Comment:
Can we move this entire `image_metadata.py` file inside `server/models/dataset_helpers/`?

Reply:
Done.